Header header

float32[3] postion 
float32 Va
float32 alpha
float32 beta

float32 chi	
float32 Vg
float32 wn
float32 we 

float32 N_dot
float32 E_dot
float32 d_dot

float32 u
float32 v
float32 w

float32 u_dot
float32 v_dot
float32 w_dot

float32 phi
float32 theta
float32 psi

float32 phi_dot
float32 theta_dot
float32 epsi_dot

float32 p
float32 q
float32 r

float32 p_dot
float32 q_dot
float32 r_dot

float32 fx
float32 fy
float32 fz

float32 l
float32 m
float32 n

float32[3] quat
bool quat_vaild
float32 chi_deg
float32 psi_deg

float32 initial_lat
float32 initial_lon
float32 initial_alt
